virttest/qemu_devices/qcontainer.py

```diff
@@ -942,6 +942,26 @@ def get_aavmf_vars(params):
                                                   parent_bus={'aobject': 'pci.0'}))
             return devices
 
+        def machine_s390(cmd=False):
```

Comment:
I suspect the story will be similar to arm. First virtio-bus version is developed and later other versions might come. For that reason I'd suggest naming this variant more-specific like `machine_s390x_virtio` or `machine_s390x_mmio`.